### DISCUSSION TITLE
Request with recaptcha v2 not filled

### DISCUSSION THREAD

**Discussion by SimTht** [emailjs-com/help#23] [Q&A] [answered]
Hello,

I am encountering the following problem : every time someone send a form without checking the recaptcha box it come back a 400 response (until there every thing is fine) but this request is taken from my quota of 200 mails per month eventho no mail was send. Is there any way to avoid this ? Except by blocking the submit from the client side if the token is empty (I think but I am not sure that a bot can bypass this kind of system).

Thanks ;)

**Comment by xr0master** [comment, marked as answer]
Since the request was sent and processed, the quota was spent accordingly.
I strongly recommend setting the validation so that the reCaptcha is required. Of course, bots can bypass this, but to do it, an attacker needs to write a special bot for your specific case. Is your site so important for attackers?

**Reply by SimTht** [reply]
Thanks for the response, that's what I have done. And no nothing worthy on this site so I will keep it this way.
